docs/tasks/stateful-sets/deleting-a-statefulset.md

```diff
@@ -0,0 +1,82 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- janetkuo
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to delete a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Deleting a StatefulSet
+
+A Stateful Set can be deleted like other resources in Kubernetes. 
```

Comment:
Address the user directly; this is in passive voice.

Expound a bit: "You can delete a StatefulSet in the same way you delete other resources in kubernetes: use the `kubectl delete` command, and specify the StatefulSet either by file or by name."